sys/include/can/can.h

```diff
@@ -95,15 +95,54 @@ struct can_frame {
     uint8_t data[CAN_MAX_DLEN] __attribute__((aligned(8)));
 };
 
+/**
+ * @brief Controller Area Network filter type
+ *
+ * The different CAN controllers provide different CAN filter
+ * types. These types are provided by the SAMD5x family (please check
+ * chapter 39.9.5 and 39.9.6 of the SAMD5x/SAME5x family Datasheet).
+ *
+ * The current CAN filter types list is from the SAMD5x/SAME5x family.
+ *
+ * Can be extended by other filter types if provided
+ */
+typedef enum {
+    CAN_FILTER_TYPE_RANGE = 0x00,   /**< Range filter from Filter 1 to Filter 2 (Filter 2 > Filter 1) */
+    CAN_FILTER_TYPE_DUAL,           /**< Dual ID Filter (Filter 2 or Filter 1) */
+    CAN_FILTER_TYPE_CLASSIC,        /**< Classic Filter: Filter ID and Mask */
```

Comment:
I still think classic should be 0 so it can be the default

Reply:
Resolved